main_test.go

```diff
@@ -76,6 +78,14 @@ func (f *ComponentTest) InitializeTestSuite(ctx *godog.TestSuiteContext) {
 }
 func TestComponent(t *testing.T) {
 	if *componentFlag {
+		log.SetDestination(io.Discard, io.Discard)
+		golog.SetOutput(io.Discard)
+		// Commented out pending fix for log.go library:
```

Comment:
I think this is now fixed (https://github.com/ONSdigital/log.go/pull/34) but would need checking...